### DISCUSSION TITLE
Add support for 'identity_smtp' plugin

### DISCUSSION THREAD

**Discussion by Meisterzunge** [Mailu/Mailu#3381] [Feature requests / Ideas]
I would **love** to have [this plugin](https://github.com/CocytusServices/Roundcube-Identity-SMTP) supported to be able to setup identities with different smtp servers than the servers default.

**Comment by Meisterzunge** [comment]
I got it working now. But that's just a workaround to get started!

I would appreciate if some dev could work on this issue to support it officially. 

# My setup

- docker deployment

# My modifications

On the **docker host** machine:
- clone the [plugin repo](https://github.com/deflomu/Roundcube-SMTP-per-Identity-Plugin) into `/mailu/overrides/roundcube`
- added the `identity_smtp` plugin to `/mailu.env` / `ROUNDCUBE_PLUGINS`

In the **docker `mailu-webmail-1` container** machine:
- create a symlink `ln -s /overrides/identity_smtp` that points to `/var/www/roundcube/plugins/identity_smtp`

# Annotations

I don't think that this is a bug.
It seems that there is just not a supported workflow for plugins that are not just added via `mailu.env`, but require to copy some files.

If i had a look at the containers `/start.py` script, I did not find any code to copy or create symlinks for plugins in the `oderrides` folder.

**Comment by o-bingo** [comment]
I also would like to use the identity_smtp plugin. Integrating worked for me, but whenever I try to send an email via external smtp-server, I get a "SMTP Error (535) Authentication failes" error in roundcube along with a submission error in the log. I tried different SMTP-servers
Is there a way to use external SMTP-servers with identity_smtp in Roundcube/mailu?
